src/angular/src/app/button-docs/button-docs.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-button-docs',
+  templateUrl: './button-docs.component.html',
+  styleUrls: ['./button-docs.component.css']
```

Comment:
We can get rid of all the .css files. If they want to add additional css overrides they can use 'style' here in the .ts file and write the code there. They wouldn't need an additional .css file.